docs/source/developer_guide/tutorial.rst

```diff
@@ -187,20 +187,34 @@ Alternatively, using ``tox``::
 Adding a new Observable
 =======================
 
-Additional built-in observable quantities may be added for use by the ``janus_core.processing.correlator.Correlation`` class. These should conform to the ``__call__`` signature of ``janus_core.helpers.janus_types.Observable``. For a user this can be accomplished by writing a function, or class also implementing a commensurate ``__call__``.
+Additional built-in observable quantities may be added for use by the ``janus_core.processing.correlator.Correlation`` class. These should extend ``janus_core.processing.observables.Observable``. The abstract method ``__call__`` should be implemented to obtain the values of the observed quantity from an ``Atoms`` object:
 
-Built-in observables are collected within the ``janus_core.processing.observables`` module. For example the ``janus_core.processing.observables.Stress`` observable allows a user to quickly setup a given correlation of stress tensor components (with and without the ideal gas contribution). An observable for the ``xy`` component is obtained without the ideal gas contribution as:
+.. code-block:: python
+
+    @abstractmethod
+    def __call__(self, atoms: Atoms) -> list[float]
+
+These values are returned as a ``list[float]`` representing the dimensions of the observed value which may be returned for a slice of atoms individually. ``value_count`` should be overloaded to return the expected length returned by ``__call__``. This is so enough correlators may be spawned to track the correlated the values. Note the final correlation will be the average of the correlations across these dimensions and atom counts.
```

Comment:
> These values are returned as a ``list[float]`` representing the dimensions of the observed value which may be returned for a slice of atoms individually.

It's not entirely clear to me what this means

Reply:
The idea is that an observable is supporting something that is computed over a set of atoms and then returns a fix dimensional value, e.g. stress, or something that also returns a value across all atoms as in velocity. Not sure how to re word it to be more clear.